### DISCUSSION TITLE
PocketCloud - New PocketBase Hosting Platform

### DISCUSSION THREAD

**Discussion by scorder96** [pocketbase/pocketbase#5833] [Show and tell]
Hi, I've been working on this project tirelessly for the past few weeks. A simple, straightforward and minimal platform to host your PocketBase instances.

The project is called PocketCloud. I am planning to improve and add features that the community wants.

Therefore, I would appreciate if people would be interested to try it out and provide feedback or criticism.

https://pocketcloud.xyz
https://discord.com/invite/P8ny2rMyms

Join the Discord to discuss! 👆

![pocketcloud-rectangle](https://github.com/user-attachments/assets/dfb19573-faf0-4959-8a63-07a2d276ca51)

**Comment by starredev** [comment]
I prefer pockethost more. What the difference?

**Reply by scorder96** [reply]
Hi, I haven't dived too deep into features that pockethost provides, so I can't provide a one-to-one comparison. But here are some features of PocketCloud -

- Direct uploading of your pb_data files, so you can continue where you left off with your development instance or to enable easy transfer of an instance hosted elsewhere.
- Direct deployment and hosting of your frontend static files (GitHub live integration coming soon). Host your complete project on PocketCloud.
- Custom domains are supported in the free plan.
- Automatic daily backups of all instances from our side.
- 3 layer redundancy system, which aims to eliminate downtime.

Moreover, PocketCloud is very new, and hence serves as a fresh canvas for the community to paint it into the platform they want.

_PS: TOTP MFA was also in my list. I see you've done some work on that. Might put it in the map if it's a desired feature by the community._

**Reply by starredev** [reply]
Thanks! What about pb hooks?

**Reply by scorder96** [reply]
I haven't personally used pb_hooks in my PocketBase projects. But I understand how powerful it is. Do you suggest enabling hook uploads through PocketCloud?

**Reply by starredev** [reply]
> I haven't personally used pb_hooks in my PocketBase projects. But I understand how powerful it is. Do you suggest enabling hook uploads through PocketCloud?

I would say yes, I use it a lot in my own projects.

it is something that can convince people to use it (including me)

**Reply by scorder96** [reply]
Hooks are live!

**Comment by ganigeorgiev** [comment]
_Side-note: I haven't checked the above platform yet but be aware that with v0.23.0 there are changes related to how the first admin/superuser account is created and if you rely that users can just visit `/_/` for the initial setup, it will no longer work and you have to either generate manually the superuser OR show somewhere the stderr logs so that users can copy/paste the generated short-lived url. For more details please check https://github.com/pocketbase/pocketbase/discussions/5814. I'll ping @benallfree because his project could be also affected by it. Nothing is set in stone yet so if anyone has a better idea how to handle it feel free to reply in the related discussion._

**Reply by benallfree** [reply]
Yes thank you we are tracking the v23 release and it definitely requires updating the logic for this stuff.

**Reply by scorder96** [reply]
Thanks for the heads-up. Will keep an eye on this.

**Comment by benallfree** [comment]
Congrats! Same question I ask everyone else: where's the open source github repo? ;)

**Reply by scorder96** [reply]
Thanks! And good point. The PocketCloud website will be open sourced once I am done with organizing and documenting the code, and going through a final review. Admittedly, it is not the highest in my to-do list since it's a busy month, but surely before Christmas.
